packages/website/src/components/Layout/components/Navbar/NavbarElements.js

```diff
@@ -172,15 +172,21 @@ export const NavSliderInfo = styled(FlexCenter)`
     display: none;
   }
 `
+export const NavbarVerticalContent = styled(FlexCenter)`
+position: fixed;
+bottom: 1rem;
+width: 5rem;
+height: 82vh;
+flex-direction: column-reverse;
+justify-content: space-between;
+padding:3rem 0;
+overflow: hidden;
+`
```

Comment:
Hey @Kanishkrawatt, can you please explain how are we using NavbarVerticalContent?

Reply:
Got it and is there a specific reason to use `82vh` only?

Reply:
By deducting the height of the top nav-bar from the height of the entire screen, the resulting value is  `approx. 82vh` represents the available vertical space.